wallet-sdk/src/main/kotlin/org/stellar/walletsdk/util/FormatOperation.kt

```diff
@@ -0,0 +1,163 @@
+package org.stellar.walletsdk.util
+
+import org.stellar.sdk.Asset
+import org.stellar.sdk.responses.operations.CreateAccountOperationResponse
+import org.stellar.sdk.responses.operations.OperationResponse
+import org.stellar.sdk.responses.operations.PathPaymentBaseOperationResponse
+import org.stellar.sdk.responses.operations.PaymentOperationResponse
+import org.stellar.walletsdk.*
+
+/**
+ * Format Stellar account operations to make them consistent.
+ *
+ * @param accountAddress Stellar address of the account
+ * @param operation Stellar operation to format
+ *
+ * @return formatted operation
+ */
+fun formatStellarOperation(accountAddress: String, operation: OperationResponse): WalletOperation {
+  when (operation.type) {
+    // Create account
+    StellarOperationTypes.CREATE_ACCOUNT.type -> {
+      val isCreator = (operation as CreateAccountOperationResponse).funder == accountAddress
+
+      return WalletOperation(
+        id = operation.id,
```

Comment:
nit: field names can be left unspecified

Reply:
In Kotlin, is it preferable not to include field names? In IDE, there are hints for that, but in GitHub, it's hard to tell what value goes where. I've been using field names to make sure I assign the correct value. :sweat_smile: